### PR TITLE
Change overDrag prop default value to false

### PR DESCRIPTION
**Pull request by DanijelBojcic**
# Summary
Changing `overDrag` prop default value to false.

https://github.com/callstack/react-native-pager-view/issues/330#issuecomment-849493733

## Compatibility

| OS      | Implemented |
| ------- | :---------: |
| iOS     |    ✅❌     |
| Android |    ✅❌     |

## Checklist

- [x] I have tested this on a device and a simulator
- [x] I added the documentation in `README.md`
- [x] I updated the typed files (TS and Flow)

### CODE REVIEW

File: ios/ReactNativePageView.m [hunk 1]
```diff
@@ -390,6 +382,16 @@ - (void)scrollViewDidScroll:(UIScrollView *)scrollView {
         absoluteOffset =  1 - absoluteOffset;
     }
     
+    if (!_overdrag) {
```
Comment (troZee) [inline ios/ReactNativePageView.m:385]: Could you update your branch with the latest master ? 
https://github.com/callstack/react-native-pager-view/pull/363/files

**Review by troZee — CHANGES_REQUESTED** [review]
Thank you for your contribution 🎉  I have added a one comment here

File: ios/ReactNativePageView.m (hunk 1 above)
Comment (DanijelBojcic) [reply ios/ReactNativePageView.m]: Done!

**Comment by troZee** [comment]
https://github.com/callstack/react-native-pager-view/releases/tag/v5.2.0

**Pull request merged**